templates/default/contentlayer.config.js

```diff
@@ -16,6 +16,7 @@ const sharedFields = {
   description: { type: 'string' },
   image: { type: 'string' },
   layout: { type: 'string', default: 'docs' },
+  editLink: { type: 'boolean', default: false},
```

Comment:
Having this default value set to false independently of the value in config, results in the need to manually change both values in order to add the link to docs pages (without the need to use frontmatter).

Reply:
If this value comes from the config, the problem is solved.

Reply:
and the config is already available in contentlayer config (imported as `siteConfig`)